### PR TITLE
[FIX] l10n_ar_withholding_ux: set valid default_type_tax_use param value in withholding tax form

### PR DESCRIPTION
**Pull request by jue-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/994).

File: l10n_ar_withholding_ux/views/account_tax_view.xml [hunk 1]
```diff
@@ -73,7 +73,7 @@
         <field name="res_model">account.tax</field>
         <field name="view_mode">tree,form</field>
         <field name="domain" eval="False"/>
-        <field name="context">{'search_default_customer': True, 'search_default_supplier': True, 'default_type_tax_use': 'supplier', 'active_test': False}</field>
```
Comment (zaoral) [inline l10n_ar_withholding_ux/views/account_tax_view.xml:76]: add other field with default supplier
and update the manifest

**Review by zaoral — CHANGES_REQUESTED** [review]

File: l10n_ar_withholding_ux/views/account_tax_view.xml (hunk 1 above)
Comment (zaoral) [reply l10n_ar_withholding_ux/views/account_tax_view.xml:76]: https://github.com/ingadhoc/odoo-argentina/blob/279ee282fc86fdfee2b52bfad857c105178857ad/l10n_ar_account_withholding/models/account_tax.py#L26

this one also change it
ask to Filo

**Comment by zaoral** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@jue-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina#1008

**Pull request closed without merging**